js/script.js

```diff
@@ -13,5 +13,231 @@ function closeMenu() {
+    demo: '',
+  },
+];
+
+// Add main Popup//
+const buttonProject = document.querySelectorAll('.btn-see-project');
+
+const mainPopup = document.createElement('div');
+mainPopup.className = 'mainPopup';
+buttonProject.forEach((element, index) => {
+  element.addEventListener('click', () => {
+    const popup = document.createElement('div');
+    popup.className = 'popup';
+    popup.innerHTML = `
+      <div>
+        <img class="imagePopup" src=${dataObj[index].image[0]} alt="image" />
+        <a href="#" id="close-btn1" onclick='closeProject()'
+          ><ion-icon name="close-outline"></ion-icon
+        ></a>
+      </div>
+      <div class="content">
+        <h2>${dataObj[index].title}</h2>
+        <div id="flex-container" class="flex-container-div">
+        <ul class="flex-container" id="flex-container">${dataObj[
+    index
+  ].card_list
+    .map((elem) => `<li class="flex-items">${elem}</li>`)
+    .join('')}   
+        </ul>
+        </div>
+        <p class="textPopup"
+          >${dataObj[index].about}</p
+        >
+      </div>
+      <div class="btnPopup"
+        ><a href="#" id="livePopup" class="btn-see-project"
+          >See Live
+          <ion-icon class="ion-icon" name="caret-forward-circle-outline"></ion-icon></a
+        ><a href="#" id="sourcePopup" class="btn-see-project"
+          >See Source <ion-icon class="ion-icon" name="logo-github"></ion-icon></a
+      ></div>
+    `;
+    mainPopup.appendChild(popup);
+  });
+});
+
+function openProject() {
+  main.appendChild(mainPopup);
+}
+function closeProject() {
+  mainPopup.style.display = 'none';
+}
+
+/// // *** POPUP WINDOW desktop *** /////
+
+const mainPopupDesktop = document.createElement('div');
+mainPopupDesktop.className = 'mainPopupDesktop';
+buttonProject.forEach((element, index) => {
+  element.addEventListener('click', () => {
+    mainPopupDesktop.innerHTML = `<div class="popupDesktop">
+    <div class="imagePopupDesktop">
+      <button class="exitBtn" onclick='closeProjectDesktop()'></button>
+      <img src=${dataObj[index].image[1]} alt="Project image" />
+    </div>
+    <div class="components">
+      <h2>${dataObj[index].title}</h2>
+      <div class="btnPopup"
+        ><a href="#" id="livePopup" class="btn-see-project"
+          >See Live
+          <ion-icon name="caret-forward-circle-outline"></ion-icon></a
+        ><a href="#" id="sourcePopup" class="btn-see-project"
+          >See Source <ion-icon name="logo-github"></ion-icon></a
+      ></div>
+    </div>
+    <div class="flex-container-div">
+      <ul class="flex-container" id="flex-container">${dataObj[index].card_list
+    .map((elem) => `<li class="flex-items">${elem}</li>`)
+    .join('')}   
+      </ul>
+    </div>
+    <p class="textPopupDesktop"
+      >${dataObj[index].about}</p
+    >
+    </div>`;
+  });
+});
+
+function openProjectDesktop() {
+  main.appendChild(mainPopupDesktop);
+}
```

Comment:
- [ ] **Required:**
- Currently as observed by the previous reviewer, this function only executes once and after, the popup doesn't work any longer. Kindly fix this.

Reply:
- I cannot view the projects section of your page, please fix this issue 🙏 
Your project:
<img width="207" alt="eefeddvdvd" src="https://user-images.githubusercontent.com/73607512/203735376-8bd31fde-7e5c-4e62-b20d-ec3cdf6d6a2e.PNG">